src/kiota/Handlers/BaseKiotaCommandHandler.cs

```diff
@@ -31,7 +31,10 @@ protected static TempFolderTokenStorageService GetGitHubPatStorageService(ILogge
         Logger = logger,
         FileName = "pat-api.github.com"
     };
-    internal static readonly HttpClient httpClient = new();
+    protected HttpClient httpClient
+    {
+        get => GetHttpCient();
```

Comment:
This change will create a new client every time the property is called. 
Can you add a backing field, and update the getter to assign the field if it's null, and then return the field's value instead please?

Reply:
Thanks for pointing this out, changes are done, please have a look